CIAOS-Rust/server/src/storage.rs

```diff
@@ -0,0 +1,155 @@
+// storage.rs
+
+use std::fs::{OpenOptions, File};
+use std::io::{self, Read, Write, Seek, SeekFrom};
+use actix_web::Error;
+use log::{error, info};
+use flatbuffers::{root, FlatBufferBuilder};
+use actix_web::error::{ErrorInternalServerError, ErrorBadRequest};
+use serde_json::json;
+
+use crate::util::Flatbuffer_Store_generated::store::{FileDataList, FileData, FileDataListArgs, FileDataArgs};
+
+
+
+
+//OpenFile provides operation functionalities over the .bin file
+struct OpenFile {
+    file: File,
+}
+
+
+
+impl OpenFile {
+    //the actual function dealing with file 
```

Comment:
Lot of spelling mistakes in the comments. Please correct and use multiline comments while describing the functions